catalog/plugin/catalog-plugin-metacardbackup-storage-common/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/common/MetacardStorageRoute.java

```diff
@@ -77,9 +78,11 @@ public void stop(int code) {
       List<RouteDefinition> routesToRemove = new ArrayList<>();
       CamelContext context = getContext();
       for (RouteDefinition routeDefinition : context.getRouteDefinitions()) {
-        context.stopRoute(routeDefinition.getId());
-        routesToRemove.add(routeDefinition);
-        setRouteCollection(new RoutesDefinition());
+        if (getRouteIds().contains(routeDefinition.getId())) {
```

Comment:
Will we benefit from adding a unit test for this condition?

Reply:
I'm not sure a unit test would do more than artificially test if .contains() matches.